### PR TITLE
Refs #33522 - fix ExclamationCircleIcon import

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: webpack/formHelper.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
   DatePicker,
   TimePicker,
 } from '@patternfly/react-core';
-import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
```
Comment (evgeni) [inline webpack/formHelper.js:10]: I have no idea why, but CI won't pass w/o the `{}`

Comment (ezr-ondrej) [reply webpack/formHelper.js:10]: it is how the JS import works... only default export can be imported without the `{}` and that can be just one per file. If you export multiple stuff from one file, you need `{}`

**Review by ezr-ondrej — APPROVED** [review]
@evgeni FTW :+1:

File: webpack/formHelper.js (hunk 1 above)
Comment (evgeni) [reply webpack/formHelper.js:10]: Another point on the "reasons Evgeni will never like JS" list

**Pull request merged**